docs-src/index.rst

```diff
@@ -31,7 +33,11 @@ The Slack platform offers several APIs to build apps. Each Slack API delivers pa
 +--------------------------------+-----------------------------------------------------------------------------------------------+------------------------------------+
 | OAuth                          | Setup the authentication flow using V2 OAuth for Slack apps.                                  | ``slack_sdk.oauth``                |
 +--------------------------------+-----------------------------------------------------------------------------------------------+------------------------------------+
-| RTM API                        | Listen for incoming messages and a limited set of events happening in Slack, using WebSocket. | ``slack_sdk.rtm```                 |
+| Audit Logs API                 | Receive audit logs API data.                                                                  | ``slack_sdk.audit_logs``           |
++--------------------------------+-----------------------------------------------------------------------------------------------+------------------------------------+
+| SCIM API                       | Utilize the SCIM APIs for provisioning and managing user accounts and groups                  | ``slack_sdk.scim``                 |
```

Comment:
```suggestion
| SCIM API                       | Utilize the SCIM APIs for provisioning and managing user accounts and groups.                  | ``slack_sdk.scim``                 |
```

Reply:
Will modify manually later (rst requires consistent indent in a table)